impl/src/main/java/org/jboss/cdi/tck/tests/full/interceptors/tests/contract/method/Fish.java

```diff
@@ -14,10 +14,12 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.jboss.cdi.tck.interceptors.tests.contract.method;
+package org.jboss.cdi.tck.tests.full.interceptors.tests.contract.method;
```

Comment:
Is this perhaps one of those cases where the test is part of a hypothetical Interceptors TCK, and this change moves it to the CDI TCK?